src/css/atoms/button.css

```diff
@@ -99,3 +103,26 @@
     text-decoration: none;
   }
 }
+
+.button-hollow:not(.button-neutral) {
+  background: transparent;
+  color: $color-default;
+
+  &:hover {
+    background: $color-default;
+    color: $color-neutral;
+  }
+
+  @each $color in primary, secondary, danger {
+
+    &.button-$(color) {
```

Comment:
i'm not sure about this point.

Using like this, we are overwriting `button-$colors` properties. I prefer to make it like an atom and have is own context, like `button-hollow-$colors`.

@fernandofleury @rands0n @brnrdog